src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -45,63 +45,77 @@ internal sealed class RegexParser
         private int[] _capnumlist;
         private List<string> _capnamelist;
 
-        private RegexOptions _options;
-        private List<RegexOptions> _optionsStack;
+        private RegexOptions _option;
```

Comment:
Can this have more than one bit set? If it is the case, `_options` is a better name for it.

Reply:
yes it usually has multiple bits set. You are right, plural makes more sense then. I'll revert my change.